core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -71,64 +72,129 @@ object ConsumerGroupCommand {
       }
     } catch {
       case e: Throwable =>
-        println("Error while executing consumer group command " + e.getMessage)
-        println(Utils.stackTrace(e))
+        consumerGroupOutputWriter.printError("Executing consumer group command failed due to " + e.getMessage
+            + "\n" + Utils.stackTrace(e))
```

Comment:
I believe this is being handled in [this other PR](https://github.com/apache/kafka/pull/1548). If it's okay I would let it come through that PR. One of the two PRs would have to be rebased depending on which one goes in first.